CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ Changes are grouped as follows
 - `Removed` for now removed features.
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
+## [7.59.1] - 2024-09-12
+### Added
+- A bug in `client.sequences.data.retrieve_dataframe(...)`.
```

Comment:
Can you be more specific? What exception was raised in what condidion?

Reply:
What do you think about the latest change?

Reply:
Looks good.